### ISSUE TITLE
broken pdf files

### ISSUE THREAD

**Issue by zvyagaaa** [GadgetAngel/BTT_SKR_13_14_14T_SD-DFU-Bootloader#15]
I can't see pdf files with a description of how to flash, github swears at the pdf - says they are damaged

**Comment by MoJo2600** [comment]
yes, sadly... I have broken skr mini e3 v3 and would like to flash the firmware. But without a Guide it's a bit hard.

**Comment by MasterAlbus3d** [comment]
should it be cloned with git lfs clone instead ??
